nix/default.nix

```diff
@@ -49,7 +73,7 @@ let
   };
 
   # pull latest nixpkgs for just [private] support
-  latestPkgs = import (pkgs.fetchFromGitHub {
+  latestPkgsForJust = import (pkgs.fetchFromGitHub {
```

Comment:
I think this can be removed, the new "latestPkgs" you just added should contain the right just version, so just removing this binding may be enough.